### PR TITLE
Revert "Update inventory.py"

### PR DESCRIPTION
**Pull request by atakanserifoglu**
This reverts commit 7ee28848e7372c3f59ece193ac98be40060850ef.

### CODE REVIEW

File: inventory.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import datetime  # Used for date and time in sale records
 import json  # For saving and loading inventory and sales data to/from JSON files
 import os  # For checking file existence when loading data
-import logging
-from typing import Optional  # For logging operations such as adding items or recording sales
+import logging  # For logging operations such as adding items or recording sales
 
 # Configure logging settings to write logs to a file with specific format
 logging.basicConfig(filename='store_log.txt', level=logging.INFO,
```
Comment (atakanserifoglu) [inline inventory.py:7]: deneme

File: inventory.py [hunk 2]
```diff
@@ -295,22 +294,6 @@ def load_data(self):
...
-    end_test_datetime = datetime.datetime.strptime('2022-06-01', '%Y-%m-%d')
-
-    if example_datetime < end_train_datetime:
-        return 'train'
-    elif example_datetime < end_valid_datetime:
-        return 'validation'
-    elif example_datetime < end_test_datetime:
-        return 'test'
-    else:
-        return None
```
Comment (atakanserifoglu) [inline inventory.py:313]: why delete this

File: inventory.py [hunk 3]
```diff
@@ -295,22 +294,6 @@ def load_data(self):
         self.inventory.load_inventory()
         self.sales_history.load_sales_history()
 
-def _temporal_split_from_datetime(
-    example_datetime: datetime.datetime
-) -> Optional[str]:
-    """Finds the split name using the example datetime."""
```
Comment (atakanserifoglu) [inline inventory.py:301]: zorti

File: inventory.py [hunk 4]
```diff
@@ -295,22 +294,6 @@ def load_data(self):
         self.inventory.load_inventory()
         self.sales_history.load_sales_history()
 
-def _temporal_split_from_datetime(
-    example_datetime: datetime.datetime
-) -> Optional[str]:
```
Comment (atakanserifoglu) [inline inventory.py:300]: en zor

File: inventory.py [hunk 5]
```diff
@@ -1,8 +1,7 @@
 import datetime  # Used for date and time in sale records
 import json  # For saving and loading inventory and sales data to/from JSON files
 import os  # For checking file existence when loading data
-import logging
-from typing import Optional  # For logging operations such as adding items or recording sales
+import logging  # For logging operations such as adding items or recording sales
```
Comment (atakanserifoglu) [inline inventory.py:4]: bulmaca

Comment (atakanserifoglu) [inline inventory.py:4]: raco

Comment (atakanserifoglu) [inline inventory.py:4]: deneme

File: inventory.py [hunk 6]
```diff
@@ -295,22 +294,6 @@ def load_data(self):
         self.inventory.load_inventory()
         self.sales_history.load_sales_history()
 
-def _temporal_split_from_datetime(
-    example_datetime: datetime.datetime
```
Comment (atakanserifoglu) [inline inventory.py:299]: hm

File: inventory.py [hunk 7]
```diff
@@ -1,8 +1,7 @@
 import datetime  # Used for date and time in sale records
 import json  # For saving and loading inventory and sales data to/from JSON files
 import os  # For checking file existence when loading data
-import logging
-from typing import Optional  # For logging operations such as adding items or recording sales
+import logging  # For logging operations such as adding items or recording sales
 
 # Configure logging settings to write logs to a file with specific format
```
Comment (atakanserifoglu) [inline inventory.py:6]: anana

File: inventory.py [hunk 8]
```diff
@@ -1,8 +1,7 @@
 import datetime  # Used for date and time in sale records
 import json  # For saving and loading inventory and sales data to/from JSON files
 import os  # For checking file existence when loading data
-import logging
-from typing import Optional  # For logging operations such as adding items or recording sales
+import logging  # For logging operations such as adding items or recording sales
```
Comment (atakanserifoglu) [inline inventory.py:5]: mmm

Comment (atakanserifoglu) [inline inventory.py:5]: anen

File: inventory.py [hunk 9]
```diff
@@ -295,22 +294,6 @@ def load_data(self):
         self.inventory.load_inventory()
         self.sales_history.load_sales_history()
```
Comment (atakanserifoglu) [inline inventory.py:296]: aaa

File: inventory.py [hunk 10]
```diff
@@ -1,8 +1,7 @@
 import datetime  # Used for date and time in sale records
 import json  # For saving and loading inventory and sales data to/from JSON files
 import os  # For checking file existence when loading data
-import logging
-from typing import Optional  # For logging operations such as adding items or recording sales
```
Comment (atakanserifoglu) [inline inventory.py:5]: k

File: inventory.py [hunk 11]
```diff
@@ -1,8 +1,7 @@
 import datetime  # Used for date and time in sale records
 import json  # For saving and loading inventory and sales data to/from JSON files
 import os  # For checking file existence when loading data
-import logging
```
Comment (atakanserifoglu) [inline inventory.py:4]: l

File: inventory.py (hunk 5 above)
Comment (atakanserifoglu) [inline inventory.py:4]: deneme

Comment (atakanserifoglu) [inline inventory.py:4]: deneme

Comment (atakanserifoglu) [inline inventory.py:4]: aa

Comment (atakanserifoglu) [inline inventory.py:4]: asa

Comment (atakanserifoglu) [inline inventory.py:4]: asasa

File: inventory.py (hunk 10 above)
Comment (atakanserifoglu) [inline inventory.py:5]: asasasas

Comment (atakanserifoglu) [inline inventory.py:5]: asa

File: inventory.py [hunk 12]
```diff
@@ -1,8 +1,7 @@
 import datetime  # Used for date and time in sale records
 import json  # For saving and loading inventory and sales data to/from JSON files
 import os  # For checking file existence when loading data
```
Comment (tech-extremists) [inline inventory.py:3]: asasa

Comment (tech-extremists) [inline inventory.py:3]: assasa

File: inventory.py (hunk 11 above)
Comment (tech-extremists) [inline inventory.py:4]: asas

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by tech-extremists — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Pull request closed without merging**